examples/app.py

```diff
@@ -53,26 +66,58 @@ def index():
     spotify = spotipy.Spotify(auth_manager=auth_manager)
     return f'<h2>Hi {spotify.me()["display_name"]}, ' \
            f'<small><a href="/sign_out">[sign out]<a/></small></h2>' \
-           f'<a href="/playlists">my playlists</a>'
+           f'<a href="/playlists">my playlists</a> | ' \
+           f'<a href="/currently_playing">currently playing</a> | ' \
+		   f'<a href="/current_user">me</a>' \
 
 
 @app.route('/sign_out')
 def sign_out():
     os.remove(session_cache_path())
     session.clear()
+    try:
+        # Remove the CACHE file (.cache-test) so that a new user can authorize.
+        os.remove(CACHE)
```

Comment:
this needs to be `os.remove(session_cache_path())`

Reply:
Done.